packages/plugin-js/src/index.ts

```diff
@@ -32,6 +32,10 @@ interface TSResult {
   modes: {[id: string]: string | TSResult['modes']};
 }
 
+function isEmptyObject(obj: object): boolean {
+  return Object.keys(obj as object).length === 0;
+}
```

Comment:
This logic is pulled from https://github.com/drwpow/cobalt-ui/blob/main/packages/core/src/util.ts#L10. I would have used that function directly, but its not exported from the core module and I didn't know if its desired to keep that API as narrow as possible.